config/application.rb

```diff
@@ -23,6 +23,7 @@
 
 module Scholarsphere
   class Application < Rails::Application
+    require 'healthchecks/healthchecks'
```

Comment:
What is 'healthchecks' ?

Reply:
eek. good catch. this is part of sidekiq monitoring that i've been working on. not sure how it snunk in here.

Reply:
No worries... this is what PR reviews are for! This kinda stuff happens to me all the time, especially when we're juggling lots of different branches.